enhancements/kubelet/oci-image-volume.md

```diff
@@ -0,0 +1,178 @@
+---
+title: oci-image-volume
+authors:
+  - bitoku
+reviewers:
+  - haircommander
+  - saschagrunert
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - mrunalp
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - None
+creation-date: 2025-05-07
+last-updated: 2025-05-07
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/OCPNODE-2929
+  - https://issues.redhat.com/browse/OCPNODE-2575
+see-also:
+  - N/A
+replaces:
+  - N/A
+superseded-by:
+  - N/A
+---
+# OCI Image Volume
+
+## Summary
+
+In Kubernetes 1.33, support for OCI Image Volume went to Beta. This allows users to package files and share them among containers in a pod without including them in the main image, thereby reducing vulnerabilities and simplifying image creation.
+
+## Motivation
+
+Supporting OCI images directly as a VolumeSource allows Kubernetes to focus on OCI standards as well as allows it to store and distribute any content using OCI registries. This allows the project to grow into use cases which go beyond running particular images.
+
+### User Stories
+
+See [KEP-4639: User Stories](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/4639-oci-volume-source#user-stories-optional)
+
+### Goals
+
+- Introduce a new VolumeSource type that allows mounting both OCI images and OCI artifacts.
```

Comment:
The goal is to enable the feature(s) for OpenShift, right? A new type would imply a new API for OpenShift only.